app/database/scripts/wellsearch/post-deploy.sql

```diff
@@ -121,6 +121,7 @@ ALTER TABLE bedrock_material_descriptor_code  SET UNLOGGED;
 ALTER TABLE casing_code                       SET UNLOGGED;
 ALTER TABLE casing_material_code              SET UNLOGGED;
 ALTER TABLE decommission_method_code          SET UNLOGGED;
+ALTER TABLE driller                           SET UNLOGGED;
```

Comment:
@DerekRoberts  we were out of sync with @Sybrand 's work that had gotten rid of it.  We had to actually add this in to get past the last error, but now that we've merged in a more recent PR we need to delete this reference to 'driller' table.